Translation/Extractor/Visitor/Translation/FormType/ChoiceExtractor.php

```diff
@@ -63,7 +63,7 @@ public function isChoiceDeclaration()
 
         return (
             $node instanceof Node\Expr\MethodCall
-            && $node->name === self::METHOD_NAME
+            && (string) $node->name === self::METHOD_NAME
```

Comment:
No need to cast to string if we are using `===` to compare with a string.

Reply:
$node->name is not a string anymore but a Variable object :+1:

Reply:
Does it have a property that we can use to avoid the magic weird behavior?